code/modules/mining/ores_coins.dm

```diff
@@ -17,6 +17,7 @@
 	var/points = 0 //How many points this ore gets you from the ore redemption machine
 	var/refined_type = null //What this ore defaults to being refined into
 	novariants = TRUE // Ore stacks handle their icon updates themselves to keep the illusion that there's more going
+	material_modifier = 0.5 //Less effective as its not refined.

```

Comment:
Make sure to test that this is low enough so that uranium ore is below `#define RAD_MINIMUM_CONTAMINATION 350`

Reply:
I solved this in my new PR by making the bags rad insulating.